server/flaskr/blueprints/auth.py

```
@@ -48,15 +50,41 @@ def login(email, password):
+    userId = data.get("user_id")
+    if userId is None:
+        return "", 401
+
+    orgId = OrganizationDataHandler.get_user_org_id(userId)
+
+    set_tenant(orgId)
+    permissions = UserDataHandler.get_user_permissions(userId=userId)
+    user = UserDataHandler.get_user_by_id(userId)
+
+    return (
+        create_auth_response(*create_tokens(userId, orgId), user, True, permissions),
```

Ruff (0.15.20)

[warning] Boolean positional value in function call

(FBT003)